gamemodes/terrortown/gamemode/server/sv_player.lua

```diff
@@ -672,12 +672,19 @@ local deathsounds = {
 }
 local deathsounds_count = #deathsounds
 
-local function PlayDeathSound(victim)
+local function PlayDeathSound(victim, isSilent)
     if not IsValid(victim) then
         return
     end
 
-    sound.Play(deathsounds[math.random(deathsounds_count)], victim:GetShootPos(), 90, 100)
+    local tbl = {
+        victim = victim,
+        sound = deathsounds[math.random(deathsounds_count)],
+    }
+
+    if not hook.Run("TTT2PlayDeathScream", tbl, isSilent) then return end
```

Comment:
I personally would check for an explicit `false` here, so a normal `if not xyz then return end` would still work. Or is this against the purpose of this PR?

Reply:
After thinking about this:
Normally I would agree, but in this case I don't think it's necessary. The Gamemode Hook would always return either false or true.

But if you think it fits better into TTT2 general code, I will change it.

Reply:
Changed this as I see that it fits the overall TTT2 style more.

RFR